snowexsql/tables/site_data.py

```diff
@@ -1,11 +1,12 @@
 from sqlalchemy import Column, Float, String
 
 from .base import Base, SingleLocationData
+from .doi import HasOneDOI
 
 
-class SiteCondition(SingleLocationData, Base):
+class SiteCondition(SingleLocationData, Base, HasOneDOI):
```

Comment:
Should the Site have the DOI or the condition?
I would vote for the Site and the conditions would be linked to a site.